Cargo.toml

```diff
@@ -17,7 +17,7 @@ This crate provides Grapheme Cluster, Word and Sentence boundaries
 according to Unicode Standard Annex #29 rules.
 """
 
-exclude = ["/.github", "/scripts", "/benches/texts"]
+exclude = ["/.github", "/benches/texts"]
```

Comment:
Actually, while you're at it, could you exclude: `*.txt`, `scripts/*.rs`. Those things we *do* want to exclude (the first is downloaded unicode data, the second is temporary rust files)